src/FormTextarea.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+
+const FormTextarea = (props) => {
+  const {controlId, label, rows, placeholder, subText, isRequired, value, handleChange} = props;
+  return (
+    <React.Fragment>
+      <Form.Group controlId={controlId}>
+        <Form.Label>{label}</Form.Label>
```

Comment:
If no `label`, perhaps don't show the component at all?

E.g. `{label && <Form.Label>{label}</Form.Label>}`